frame/treasury/src/benchmarking.rs

```diff
@@ -93,8 +93,8 @@ fn create_tips<T: Trait>(t: u32, hash: T::Hash, value: BalanceOf<T>) -> Result<(
 
 // Create proposals that are approved for use in `on_initialize`.
 fn create_approved_proposals<T: Trait>(n: u32) -> Result<(), &'static str> {
-	for i in 0 .. n {
-		let (caller, value, lookup) = setup_proposal::<T>(i);
+	for _ in 0 .. n {
+		let (caller, value, lookup) = setup_proposal::<T>(0);
```

Comment:
why setting up only the same proposal from caller 0 to beneficiary 0 ? then in spend_funds funds are all unreserved from caller 0 and are all given to beneficiary 0. But in real life proposal are likely to be different no ?

Reply:
I think there is some requirement of the signing account must from call 0.

@shawntabrizi can give a better answer.

Reply:
Looks like https://github.com/paritytech/substrate/pull/6815 should address this.

Reply:
You can undo this change for now, and have varying accounts. It may overcount a read, but we can fix it later

Reply:
Better to overcount than undercount

Reply:
As far as I see those setup proposal are to be spend in spend_funds which is in on_initialize, thus there is actually no caller and all read/write should be counted.